vkcs/kubernetes/resource_node_group.go

```diff
@@ -260,7 +268,6 @@ func resourceKubernetesNodeGroupCreate(ctx context.Context, d *schema.ResourceDa
 		return diag.Errorf(
 			"error waiting for vkcs_kubernetes_cluster %s to become ready: %s", s.ClusterID, err)
```

Comment:
By the way, I don't like how we mix statuses here, it's a bit misleading because users will think it's a cluster status when they get an error. Let's add `NODE_GROUP_` prefix to constants defining statuses for a node group.

Reply:
upd. It's not needed if we use 2 different functions.

Reply:
Rewrote to use separate wait function for node group